shared/components/Create.js

```diff
@@ -302,8 +320,31 @@ class Create extends Component {
     })
   }
 
-  handleAddExistingLocation() {
-    //todo: implement adding photo to an existing location
+  handleAddExistingLocation(itemKey, userKey, caption, image) {
+    const data = JSON.stringify({
+      itemKey: itemKey,
+      userKey: userKey,
+      caption: caption,
+      image: image
+    });
+
+    const address =`${HTTP}${SERVER_ADDR}${ENDPOINT_IMAGE}`;
+    fetch(address, {
+      method: 'POST',
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+      },
+      body: data
+    })
+    .then((response) => response.json())
```

Comment:
You should check whether response contain error or not.
If response contain error property, It was a fail post
Check out APIDoc
https://goober.herokuapp.com/docs/#api-Image-addAnImage